src/content/docs/en/guides/markdown-content.mdx

```diff
@@ -494,16 +496,18 @@ export default defineConfig({
 
 #### Customizing a plugin
 
-In order to customize a plugin, provide an options object after it in a nested array. The example below adds the [`behavior` option to the `rehype-autolink-headings` plugin](https://github.com/rehypejs/rehype-autolink-headings#optionsbehavior) in order to add the anchor tag after the headline text.
+In order to customize a plugin, provide an options object after it in a nested array. 
```

Comment:
```suggestion
In order to customize a plugin, provide an options object after it in a nested array. 

```
I tend to like a line space between a general instruction/description and showing a specific example. Up to you!

(Aside: if you really did not want a line space, we should discuss/decide then add to our writing guide guidance re: putting one sentence per line with the intention that they still show up in the same paragraph. I'm not actually a fan, as I find it distracting to read/edit the code, despite knowing that it's a "thing" that people do for referencing lines. But, this can be up for discussion!)

Otherwise, LGTM!